src/v/container/fragmented_vector.h

```diff
@@ -52,21 +52,24 @@ class fragmented_vector {
 
     // calculate the maximum number of elements per fragment while
     // keeping the element count a power of two
-    static constexpr size_t calc_elems_per_frag(size_t esize) {
-        size_t max = fragment_size_bytes / esize;
+    static constexpr size_t calc_elems_per_frag() {
+        size_t max = fragment_size_bytes / sizeof(T);
```

Comment:
was the issue constexpr? i wonder if the compiler would be happier with consteval?

Reply:
Yeah thanks, consteval is better - I also wanted to just simplify this method.